### DISCUSSION TITLE
24GHz mmWave Sleep Breathing Monitoring | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#537] [Q&A]
# 24GHz mmWave Sleep Breathing Monitoring | Seeed Studio Wiki

24GHz mmWave Sleep Breathing Monitoring

https://wiki.seeedstudio.com/Radar_MR24BSD1/

**Comment by taunoe** [comment]
How to I send data to radar with XIAO board? When I want to read device ID etc.

**Reply by xzxcessarr** [reply]
Hello, thanks for feedback! I do not know which ID of the device do you want? Or you mean the address?

**Reply by taunoe** [reply]
According to the [User Manual](https://files.seeedstudio.com/wiki/mmWave-radar/MR24BSD1_User_Manual.pdf) 8.2 there should be a possibility to read Device ID, Software version, Hardware version etc. I tried to implement the code but unsuccessfully.

**Reply by xzxcessarr** [reply]
Hello, aftering receiving your feedback, we have test the code which we providing on here: https://wiki.seeedstudio.com/Radar_MR24BSD1/#demo-4-sending-data-to-radar, we found the C code is well, and you should delete the 0x char and sent to the module by comtool or other serial tools you like, such as the code generate by the C code: 55 07 00 01 01 01 cc cc cb 61, you will receive the feedback about the device id from the device: i6uIag1tfhehh in ASCII receive mode
